api/envoy/config/route/v3/route_components.proto

```diff
@@ -1385,6 +1385,10 @@ message RateLimit {
 
       // The key to use in the descriptor entry.
       string descriptor_key = 2 [(validate.rules).string = {min_bytes: 1}];
+
+      // The optional parameter which determines if the RLS is called if a request
```

Comment:
If set to true, Envoy skips the descriptor while calling rate limiting service when header is not present in the request. Defaults to false, which skips calling rate limiting service if this header is not present in the request.